guides/faq.rst

```diff
@@ -46,7 +46,10 @@ I can't get flashing over USB to work.
 --------------------------------------
 
 ESPHome depends on the operating system the tool is running on to recognize
-the ESP. This can sometimes fail (driver missing, inside docker container, ...).
+the ESP. This can sometimes fail. Common causes are that you did not install 
+the drivers (see note below) or you are trying to upload from a Docker container 
```

Comment:
I changed this to reference the note below, there are actually two types of chips with their own drivers each, so the note below is more accurate.